mempool/src/pool/store.rs

```diff
@@ -123,6 +123,32 @@ impl MempoolStore {
         self.txs_by_id.get(&id.get())
     }
 
+    pub(super) fn assert_valid(&self) {
+        #[cfg(test)]
+        self.assert_valid_inner()
+    }
+
+    #[cfg(test)]
+    pub(super) fn assert_valid_inner(&self) {
```

Comment:
I think this can just be private if I'm not mistaken.

Reply:
Good catch, thanks.